### PR TITLE
Pie chart sample

### PR DESCRIPTION
**Pull request by kbrdsk**
https://user-images.githubusercontent.com/52506986/109042136-b0dd8400-7684-11eb-987a-6c000fa6e3c7.mp4

### CODE REVIEW

**Comment by jbaldo** [comment]
Awesome! Thanks for the video. That helps a lot. For the tooltip, it looks like they have a `tooltip` prop that takes a function. It's not clear to me what gets passed to it, but this storybook example makes it look like you at least get the id, value, color. https://nivo.rocks/storybook/?path=/story/pie--custom-tooltip

Maybe you could pass a version of your tooltip function and not have to worry about the mouse coordinates. This is what I used for the sankey chart but it seems like each component is slightly different. https://github.com/openoakland/openousd-site/blob/master/src/components/sankey-chart.js#L51

File: src/components/pie-chart.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react"
+
+import { ResponsivePie } from "@nivo/pie"
+
+import "./pie-chart.scss"
+
+const COLORS = [
```
Comment (jbaldo) [inline src/components/pie-chart.js:7]: So does Nivo not handle the colors? Or it doesn't animate them when you hover? I haven't done anything custom with color. I've only used the default Nivo themes, so not sure. Are you basically setting a different transparency when the node is active?

Comment (kbrdsk) [reply src/components/pie-chart.js:7]: Yea, I wasn't able to get it to animate on hover so I'm setting the colors for each node on every re-render in order to adjust the transparency. I think Pie handles colors in a way similar to SankeyChart - you can set a scheme in the colors property, and I copied one of those for this.

I have a sense that I'm overlooking something that's blocking both the color animations and the tooltip...

**Comment by kbrdsk** [comment]
> Awesome! Thanks for the video. That helps a lot. For the tooltip, it looks like they have a `tooltip` prop that takes a function. It's not clear to me what gets passed to it, but this storybook example makes it look like you at least get the id, value, color. https://nivo.rocks/storybook/?path=/story/pie--custom-tooltip
> 
> Maybe you could pass a version of your tooltip function and not have to worry about the mouse coordinates. This is what I used for the sankey chart but it seems like each component is slightly different. https://github.com/openoakland/openousd-site/blob/master/src/components/sankey-chart.js#L51

Yea, I tried this originally but it didn't show, I'll keep playing with it.

**Comment by kbrdsk** [comment]
Using Pie 0.62 instead of 0.67 now making it more consistent with the other packages we're using from nivo and mocked fetching data and content for the chart. Not sure where the data for OUSD total spending should go.

**Pull request closed without merging**